internal/pkg/config/fieldtags_test.go

```diff
@@ -30,41 +30,41 @@ func TestFieldTagsIdentification(t *testing.T) {
 
 	cases := []struct {
 		desc string
-		key  string
-		val  string
+		tag  string
 		want bool
 	}{
 		{
 			"built-in field tag",
-			"levee",
-			"source",
+			"`levee:\"source\"`",
 			true,
 		},
 		{
 			"custom field tag",
-			"example",
-			"sensitive",
+			"`example:\"sensitive\"`",
 			true,
 		},
 		{
 			"different tag key",
-			"notexample",
-			"sensitive",
+			"`notexample:\"sensitive\"`",
 			false,
 		},
 		{
 			"different tag value",
-			"example",
-			"notsensitive",
+			"`example:\"notsensitive\"`",
 			false,
 		},
+		{
+			"escaped tag accepted",
+			`"levee:\"source\""`,
+			true,
+		},
 	}
```

Comment:
I think it would be good to have a test case that covers the `for _, v := range strings.Split(val, ",") {` code, now that the logic for multi-value tags is in config.